src/css/infirma.scss

```diff
@@ -1,343 +1,343 @@
-:root {
```

Comment:
I didn't mean to commit these commented out styles. I was just looking at whether they were necessary—I don't believe so, but I wasn't going to remove them...yet.